sql/character.anticheat.sql

```diff
@@ -45,6 +45,18 @@ CREATE TABLE IF NOT EXISTS `players_reports_status` (
   PRIMARY KEY (`guid`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+-- Dumping structure for table characters.lua_cheaters
+DROP TABLE IF EXISTS `lua_cheaters`;
```

Comment:
Please remove this so it errors out if someone tries to run it again, so they don't lose the data

Reply:
https://github.com/TrinityCore/TrinityCoreCustomChanges/commit/1d1cdda4d4a033b99dde03f0de4606956be0f989